Design/UseCases.md

```diff
@@ -0,0 +1,46 @@
+## Actors
+* **Software System** - Payment management, Keyword analysis, Pitch cataloging, Collect statistical data.
+
+* **Founder** - Pitch creation, Create statistics.
+
+* **Investor** -Observation of parties, provide funds.
+
+* **Developer** - Completion of tasks, observation of parties.
+
+* **User (Founder, Investor, and Developer)** - Profile creation, Sending direct messages, Negotiate terms, Fluid role choice.
+## Use Cases
+* **UC1** - We want to be able to find other projects as investors and developers using keywords and similar interests.
+
+  * Search for projects. [Developer, Investor, System]
```

Comment:
These bullet points would be the use cases. They each need to be labeled as UC[#].